cookbook/bundles/best_practices.rst

```diff
@@ -28,33 +28,29 @@ the book and cookbook.
 Bundle Name
 -----------
 
-A bundle is also a PHP namespace. The namespace must follow the technical
-interoperability `standards`_ for PHP 5.3 namespaces and class names: it
-starts with a vendor segment, followed by zero or more category segments, and
-it ends with the namespace short name, which must end with a ``Bundle``
-suffix.
+A bundle is also a PHP namespace. The namespace must follow the `PSR-0`_ or
```

Comment:
no, bundle is in fact just a namespace or a name to refer to a location on the filesystem